docs/time.md

```diff
@@ -200,6 +210,23 @@ time.add({ hours: 1 }).with({
 }); // => 20:00
 ```
 
+### time.**withCalendar**(_calendar_: object | string) : Temporal.Time
+
+**Parameters:**
+
+- `calendar` (`Temporal.Calendar` or plain object or string): The calendar into which to project `time`.
+
+**Returns:** a new `Temporal.Time` object which is the time indicated by `time`, projected into `calendar`.
+
+This method is the same as `date.with({ calendar })`, but may be more efficient.
```

Comment:
s/date/time/ (and also consider removing this sentence because it seems likely that the resolution of the with() issue will be that with() will stop accepting calendars)

Reply:
Removed.